src/main/kotlin/edu/carroll/polyominoes/service/account/model/SecurityAccount.kt

```diff
@@ -0,0 +1,81 @@
+package edu.carroll.polyominoes.service.account.model
+
+import edu.carroll.polyominoes.jpa.model.Account
+import org.springframework.security.core.GrantedAuthority
+import org.springframework.security.core.userdetails.UserDetails
+
+class SecurityAccount(private val account: Account) : UserDetails {
+
+    /**
+     * Returns the username used to authenticate the user. Cannot return
+     * `null`.
+     * @return the username (never `null`)
+     */
+    override fun getUsername(): String {
+        return account.username
+    }
+
+    /**
+     * Returns the password used to authenticate the user.
+     * @return the password
+     */
+    override fun getPassword(): String {
+        return account.hashPassword
+    }
+
+    /**
+     * Returns the authorities granted to the user. Cannot return `null`.
+     * @return the authorities, sorted by natural key (never `null`)
+     */
+    override fun getAuthorities(): MutableCollection<out GrantedAuthority> {
+        return HashSet<GrantedAuthority>()
+    }
+
+    /**
+     * Indicates whether the user's account has expired. An expired account cannot be
+     * authenticated.
+     * @return `true` if the user's account is valid (ie non-expired),
+     * `false` if no longer valid (ie expired)
+     */
+    override fun isAccountNonExpired(): Boolean {
```

Comment:
Boo Spring Security isBlahNonBletch!

Who uses a not negative for indicating positive behavior?